index.js

```diff
@@ -49,7 +49,15 @@ class CircuitBreaker extends EventEmitter {
         this.shouldRetry = (options && options.shouldRetry) || (() => true);
 
         this.breaker = circuitbreaker(this.command, this.breakerOptions);
-        this.breaker.on('open', () => self.emit('open'));
+        this.breaker.on('open', () => {
+            const breakerError = ['Breaker with function',
+                self.command.toString(),
+                'was tripped on',
+                new Date().toUTCString()].join(' ');
+
+            console.log(breakerError);
```

Comment:
Use `console.error` instead. There's no reason to create a constant for the error string, use a template string.

```
console.error(`Breaker with function $(this.command.toString()) was tripped on $(new Date().toUTCString())`);
```

You are using arrow functions, so `this` should be the same context of `this` in the constructor. You don't need `self`.